File: app/src/main/java/org/rivchain/cuplink/AboutActivity.java
package org.rivchain.cuplink;

import android.content.Intent;
import android.os.Bundle;
import android.widget.TextView;


public class AboutActivity extends MeshengerActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);

        setTitle(getResources().getString(R.string.menu_about));

        ((TextView) findViewById(R.id.versionTv)).setText(
                Utils.getApplicationVersion(this)
        );

        findViewById(R.id.licenseTV).setOnClickListener(v -> {
            Intent intent = new Intent(this, LicenseActivity.class);
            startActivity(intent);
        });
    }
}

File: app/src/main/java/org/rivchain/cuplink/AboutActivity.kt
package org.rivchain.cuplink

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.TextView

class AboutActivity : CupLinkActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about)
        title = resources.getString(R.string.menu_about)
        (findViewById<View>(R.id.versionTv) as TextView).text = Utils.getApplicationVersion(this)
        findViewById<View>(R.id.licenseTV).setOnClickListener { v: View? ->
            val intent = Intent(this, LicenseActivity::class.java)
            startActivity(intent)
        }
    }
}